### PR TITLE
PURCHASE-1198 - Adds fair and museums to Search results tabs

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/PURCHASE-1198

- Adds `Fairs` and `Institutions` to search result tabs

- Depends on https://github.com/artsy/reaction/pull/2589

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 8bae5d7bca4e0ef37dce6ef8612673bb95852d50

https://deploy-preview-2548--artsy-reaction.netlify.com

File: src/Apps/Search/Components/NavigationTabs.tsx [hunk 1]
```diff
@@ -93,13 +89,16 @@ export class NavigationTabs extends React.Component<Props> {
 
     const tabCountMap = Object.entries(TAB_NAME_MAP).reduce(
```
Comment (pepopowitz) [inline src/Apps/Search/Components/NavigationTabs.tsx]: This function looks ideal for covering with unit tests. I _think_ I understand the changes you made, and why you made them, but `array.reduce` operations are (1) easy to mess up, and (2) easy to write tests for.

**Comment by kierangillen** [comment]
Moving this to WIP as we define a more concrete solution for overflowing tabs.

File: src/Apps/Search/Components/NavigationTabs.tsx (hunk 1 above)
Comment (kierangillen) [reply src/Apps/Search/Components/NavigationTabs.tsx]: Hey just updated these tests after our pairing session, should be good to merge!

**Comment by artsyit** [comment]
:rocket: PR was released in v16.18.2 :rocket:

**Pull request merged**